pom.xml

```diff
@@ -14,18 +14,18 @@
   </licenses>
 
   <properties>
-    <aspectj.version>1.9.6</aspectj.version>
+    <aspectj.version>1.9.7</aspectj.version>
     <jsonschema_paths>acq-models/mod-invoice-storage/schemas,raml-util/schemas,acq-models/common/schemas/</jsonschema_paths>
-    <raml-module-builder.version>33.0.0</raml-module-builder.version>
-    <vertx.version>4.1.0.CR1</vertx.version>
+    <raml-module-builder.version>34.1.0</raml-module-builder.version>
+    <vertx.version>4.3.1</vertx.version>
     <junit.version>4.13.1</junit.version>
-    <rest-assured.version>4.3.0</rest-assured.version>
+    <rest-assured.version>5.1.1</rest-assured.version>
     <generated_sources_dir>${project.build.directory}/generated-sources</generated_sources_dir>
     <jsonschema2pojo_output_dir>${generated_sources_dir}/jsonschema2pojo</jsonschema2pojo_output_dir>
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <mod-configuration-client.version>5.4.0</mod-configuration-client.version>
-    <spring.version>5.2.6.RELEASE</spring.version>
+    <spring.version>5.3.20</spring.version>
```

Comment:
Current spring version is 5.3.21: https://mvnrepository.com/artifact/org.springframework/spring-context